### PR TITLE
ADD the Apply configuration group management permissions and Apply in…

### PR DESCRIPTION
**Pull request by zsl1549**

### CODE REVIEW

File: src/pages/Group/Index.js [hunk 1]
```diff
@@ -607,17 +609,17 @@ class Main extends PureComponent {
             <div className={styles.connect_Box}>
               <div className={styles.connect_Boxs}>
                 <div>使用内存</div>
-                <div>{resources.memory || 0}M</div>
+                <div>{`${sourceUtil.unit(resources.memory || 0, 'MB')}`}</div>
```
Comment (barnettZQG) [inline src/pages/Group/Index.js:612]: It requires intelligent processing of data units, MB or GB

File: src/pages/Group/Index.js [hunk 2]
```diff
@@ -607,17 +609,17 @@ class Main extends PureComponent {
...
               </div>
               <div className={styles.connect_Boxs}>
                 <div>使用CPU</div>
-                <div>{resources.cpu || 0}M</div>
+                <div>{(resources.cpu && resources.cpu / 1000) || 0}核</div>
```
Comment (barnettZQG) [inline src/pages/Group/Index.js:616]: unit： Core

File: src/pages/Group/Index.js [hunk 3]
```diff
@@ -607,17 +609,17 @@ class Main extends PureComponent {
...
               </div>
             </div>
             <div className={styles.connect_Box}>
               <div className={styles.connect_Boxs}>
                 <div>使用磁盘</div>
-                <div>{resources.disk || 0}MB</div>
+                <div>{`${sourceUtil.unit(resources.disk || 0, 'MB')}`}</div>
```
Comment (barnettZQG) [inline src/pages/Group/Index.js:622]: Same problem as memory.

File: src/pages/Group/Index.js (hunk 1 above)
Comment (zsl1549) [reply src/pages/Group/Index.js:612]: Intelligent processing unit of data  sourceUtil.unit  methods

File: src/pages/Group/Index.js (hunk 3 above)
Comment (zsl1549) [reply src/pages/Group/Index.js:622]: Intelligent processing unit of data sourceUtil.unit methods

File: src/pages/Group/Index.js (hunk 2 above)
Comment (zsl1549) [reply src/pages/Group/Index.js]: After repair

**Review by barnettZQG — APPROVED** [review]

**Pull request merged**